templates/secret.yaml

```diff
@@ -10,21 +10,27 @@ metadata:
     "helm.sh/resource-policy": no-upgrade-existing
 {{- end }}
 type: Opaque
+{{- $secretName := (include "retool.fullname" .) }}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace $secretName }}
 data:
   license-key: {{ .Values.config.licenseKey | default "" | b64enc | quote }}
 
   {{ if not .Values.config.jwtSecretSecretName }}
   {{ if .Values.config.jwtSecret }}
   jwt-secret: {{ .Values.config.jwtSecret | b64enc | quote }}
-  {{ else  }}
+  {{ else if and $secret (index $secret.data "jwt-secret") }}
```

Comment:
why is and needed in this syntax :O

Reply:
i think these are like the go functions where and/or operate on the next two arguments. i was using it here as a null check, so like if the secret exists and it has a data field - i was getting some nil complaints on `helm lint` without this but there might be a better way to do it